lib/improver/spotdata/common_functions.py

```diff
@@ -347,6 +356,8 @@ def apply_bias(vertical_bias, dzs):
         available, otherwise it returns the whole set.
 
     """
+    dzs = np.array(dzs)
```

Comment:
Given that you've declared that it should be a numpy.array, is it worth a conversion? Maybe the things that call this could do it there instead?

In the future, we can use type hints: https://docs.python.org/3/library/typing.html